services/wfm/src/main/java/org/openkilda/wfm/topology/cache/CacheTopology.java

```diff
@@ -100,11 +102,17 @@ public StormTopology createTopology() throws NameCollisionException {
                 .shuffleGrouping(BOLT_ID_CACHE, StreamType.TPE.toString());
 
         /*
-         * Sends cache dump and reroute requests to `flow` topology.
+         * Sends cache dump and reroute requests to `flow` topology via throttling bolt.
          */
+        FlowThrottlingBolt flowThrottlingBolt = new FlowThrottlingBolt(
+                topologyConfig.getRerouteThrottlingMinDelay(), topologyConfig.getRerouteThrottlingMaxDelay());
+        int newParallelism = topologyConfig.getNewParallelism();
+        builder.setBolt(BOLT_ID_REROUTE_THROTTLING, flowThrottlingBolt, newParallelism)
+                .fieldsGrouping(BOLT_ID_CACHE, StreamType.WFM_REROUTE.toString(), new Fields(CacheBolt.FLOW_ID_FIELD));
```

Comment:
Separate bolt instances don't share the state of ExtendableTimeWindow. And the grouping by flowId means that sliding of the window in one instance will not affect the window in another instance. But as per requirements we should wait for all ISLs to be settled down.